bin/purge-dev-files.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I'm fine with this while we experiment, but let's figure out how to consolidate ASAP. This script will need to have a couple of functions since we do want `to-test.md` for development builds.

Reply:
Another approach I was thinking of exploring: 

Instead of deleting dev files, we build a process that adds production files to It’s own directory. It might be a pattern that will scale more easily if we end up needing custom Jetpack builds.